pkg/kubectl/cmd/create_role.go

```diff
@@ -50,20 +53,30 @@ type CreateRoleOptions struct {
 	Verbs         []string
 	Resources     []schema.GroupVersionResource
 	ResourceNames []string
+
+	DryRun       bool
+	OutputFormat string
```

Comment:
I agree, those are just common command options, not specific to this one. Do we have plan now, please let me know if I can help, thanks!